x-pack/plugins/security_solution/common/endpoint/generate_data.ts

```diff
@@ -1298,7 +1298,7 @@ export class EndpointDocGenerator {
       title: 'Elastic Endpoint',
       version: '0.5.0',
       description: 'This is the Elastic Endpoint package.',
-      type: 'solution',
+      type: 'integration',
```

Comment:
`integration` is the [only valid `type`](https://github.com/elastic/package-spec/blob/3b4d8207554dde6ab9646567dd39b747a2ce167c/versions/1/manifest.spec.yml#L45-L49) value